parsl/executors/taskvine/manager.py

```diff
@@ -44,11 +44,17 @@ def _set_manager_attributes(m, config):
     # Enable peer transfer feature between workers if specified
     if config.enable_peer_transfers:
         m.enable_peer_transfers()
+    else:
+        m.disable_peer_transfers()
```

Comment:
is this an unrelated bugfix?

Reply:
It is arguably related. This feature is on by default, so default behavior has not changed. However previously a user was not able to "disable" peer transfers. This small fix, and the addition of vine_tune is allowing the same configuration abilities you would get using the base taskvine api

Reply:
I guess the comment in the code there is a bit misleading to suggest it is not on by default